man/org.freedesktop.systemd1.xml

```diff
@@ -4852,6 +4856,14 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       <varname>StateDirectory</varname>, <varname>CacheDirectory</varname> and <varname>LogsDirectory</varname>,
       which will create a symlink of the given name to the respective directory. The messages take an unused
       <varname>flags</varname> parameter, reserved for future backward-compatible changes.</para>
+
+      <para><varname>ExtraFileDescriptorNames</varname> contains file descriptor names passed to the service via
+      the ExtraFileDescriptors property in the StartTransient dbus API. The file descriptor names are passed to
```

Comment:
enclose ExtraFileDescriptions in `<varname>`

and enclose `StartTransient` with `<function>`

Reply:
dbus → D-Bus